lib/fluent/plugin_helper/server.rb

```diff
@@ -709,11 +709,11 @@ def try_tls_accept
                 return true
               end
             rescue Errno::EPIPE, Errno::ECONNRESET, Errno::ETIMEDOUT, Errno::ECONNREFUSED, Errno::EHOSTUNREACH => e
-              @log.trace "unexpected error before accepting TLS connection", error: e
+              @log.trace "unexpected error before accepting TLS connection", peer: @_handler_socket.io.peeraddr, error: e
```

Comment:
Is `.io` needed?

I thought that `@_handler_socket.peeraddr` is enough.
(OpenSSL::SSL::SSLSocket inherit it from OpenSSL::SSL::SocketForwarder#peeraddr)

Reply:
You are right, thanks!
BTW `peeraddr` is an array like `["AF_INET", 34772, "127.0.0.1", "127.0.0.1"]`.
It's too verbose, only host and port is needed.